src/frontend/src/lib/components/tokens/TokenSymbol.svelte

```diff
@@ -0,0 +1,15 @@
+<!-- source: DFINITY foundation -->
```

Comment:
This can be removed

Reply:
Yes... not really sure how it ended up there... thank you!